### ISSUE TITLE
Option to register

### ISSUE THREAD

**Issue by hrj** [fosstr/rsvp#7]
IMO, registration can be made an optional feature.

The benefits for a user who has registered would be:
1. Email Notifications of upcoming events.
2. Email Reminder for an event that they are attending.
3. No need to enter details and captcha everytime they need to RSVP.

**Comment by hiteshd** [comment]
I thought the registration and login was going to be future enhancements and was not needed for public release?

**Comment by hrj** [comment]
Yeah; my bad. I created the issues first, then setup examples milestones.

Moved to version 0.3.

**Comment by hrj** [comment]
Feel free to move the other issues around, based on what is feasible for you and what needs to be done.

**Comment by digen** [comment]
Refreshing a discussion we've had previously on this topic, IMHO, instead of storing and managing user data at our end, OAuth(github, Stackexchange, Reddit) will save us from a lot of work and security hassles. Let's discuss this on IRC sometime soon.

**Comment by hiteshd** [comment]
+1 for OAuth, Stackoverflow, Github access. Also, For newcomers, I would also add Google. (Just the way PyBelgaum did)

**Comment by gsama** [comment]
+1 for Github. Because we expect all fosstr members to be on github anyway.
